src/kosa_crux/entity/pali_word/spec.clj

```diff
@@ -2,31 +2,31 @@
   (:require [clojure.string]
             [clojure.spec.alpha :as s]))
 
-(s/def :entity/truthy-string (s/and string? #(or (= "true" %)
+(s/def ::truthy-string (s/and string? #(or (= "true" %)
                                                  (= "false" %))))
-(s/def :entity/truthy-value boolean?)
+(s/def ::truthy-value boolean?)
 
-(s/def :pali-word/published-at inst?)
-(s/def :pali-word/bookmarkable (s/or :entity/truthy-value :entity/truthy-string))
-(s/def :pali-word/shareable (s/or :entity/truthy-value :entity/truthy-string))
-(s/def :pali-word/card-type (s/and string? #(= "pali_word" %)))
-(s/def :pali-word/pali (s/and string? #(-> % clojure.string/blank? not)))
-(s/def :pali-word/id uuid?)
-(s/def :pali-word/header
+(s/def ::published-at inst?)
+(s/def ::bookmarkable (s/or :entity/truthy-value :entity/truthy-string))
+(s/def ::shareable (s/or :entity/truthy-value :entity/truthy-string))
+(s/def ::card-type (s/and string? #(= "pali_word" %)))
+(s/def ::pali (s/and string? #(-> % clojure.string/blank? not)))
+(s/def ::id uuid?)
+(s/def ::header
   (s/and string? #(-> % clojure.string/blank? not)))
 
-(s/def :pali-word.audio/url (s/and string? #(-> % clojure.string/blank? not)))
-(s/def :pali-word/audio (s/keys :req-un [:pali-word.audio/url]))
+(s/def ::url (s/and string? #(-> % clojure.string/blank? not)))
+(s/def ::audio (s/keys :req-un [:pali-word.audio/url]))
 
-(s/def :translations/language (s/and string? #(contains? #{"hi" "en"} %)))
-(s/def :translations/translation (s/and string? #(-> % clojure.string/blank? not)))
-(s/def :translations/id uuid?)
-(s/def :pali-word/translations
+(s/def ::language (s/and string? #(contains? #{"hi" "en"} %)))
+(s/def ::translation (s/and string? #(-> % clojure.string/blank? not)))
+(s/def ::id uuid?)
+(s/def ::translations
   (s/coll-of (s/keys :req-un [:translations/id
                               :translations/translation
                               :translations/language])))
```

Comment:
I think the above block will actually break. `:translations/language`, for example, no longer exists if the registry entry is changed to `::translations` since that will auto-resolve to `:entity.pali-word.spec/translations` now.